### PR TITLE
feat: opBNB farm

### PR DESCRIPTION
**Pull request by ChefJerry**
copilot:all

### CODE REVIEW

File: apis/farms/src/v3.ts [hunk 1]
```diff
@@ -48,6 +48,10 @@ export const V3_SUBGRAPH_CLIENTS = {
   [ChainId.BASE]: new GraphQLClient('https://api.studio.thegraph.com/query/45376/exchange-v3-base/version/latest', {
     fetch,
   }),
+  // @todo @Chef-Jerry add opbnb subgraph
+  [ChainId.OPBNB]: new GraphQLClient('https://opbnb-mainnet-graph.nodereal.io/subgraphs/name/pancakeswap/exchange-v3', {
```
Comment (chefjackson) [inline apis/farms/src/v3.ts:52]: subgraphs are also exported from chains pkg. Let's import from chains

File: packages/multicall/src/constants/gasLimit.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ export const DEFAULT_GAS_LIMIT_BY_CHAIN: { [key in ChainId]?: bigint } = {
   [ChainId.ZKSYNC]: 500000000n,
   [ChainId.POLYGON_ZKEVM]: 1500000n,
   [ChainId.BASE]: 60000000n,
+  [ChainId.OPBNB]: 100_000_000n,
+  [ChainId.OPBNB_TESTNET]: 100_000_000n,
```
[the remark this answers is not in the document]
Comment (ChefJerry) [reply packages/multicall/src/constants/gasLimit.ts:11]: But as opBNB docs reference, the max gasLimit is 100M, our default max gasLimit is 150M

[the remark this answers is not in the document]
Comment (ChefJerry) [reply packages/multicall/src/constants/gasLimit.ts:11]: https://docs.bnbchain.org/opbnb-docs/docs/faq/gas-and-fees-faqs#what-is-the-block-gas-limit-on-opbnb

[the remark this answers is not in the document]
Comment (chefjackson) [reply packages/multicall/src/constants/gasLimit.ts:11]: hmm alright then let's keep it

**Review by chefjackson — APPROVED** [review]

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**